src/components/Seat/JetsSeat.integration.test.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { MockJetsContextProvider } from '../../__mocks__/MockJetsContext';
+
+import { paramsData, seatDataFirst } from './__fixtures__';
+import { JetsSeat } from './index';
+
+const setup = ({ data = {}, config = {}, params = {}, events = {} } = {}) => ({
+  user: userEvent.setup(),
+  ...render(
+    <MockJetsContextProvider config={config} events={events} params={{ ...paramsData(params) }}>
+      <JetsSeat data={data} />
+    </MockJetsContextProvider>
+  ),
+});
+
+describe('JetsSeat', () => {
+  describe('when any seat type is rendered', () => {
+    it('should not trigger handlers to scroll to the seat by default', () => {
+      const mockScrollIntoView = jest.fn();
+      const resetSeatJumpTo = jest.fn();
+      const showTooltip = jest.fn();
+
+      window.HTMLElement.prototype.scrollIntoView = mockScrollIntoView;
+
+      setup({
+        data: seatDataFirst(),
+        events: { resetSeatJumpTo, showTooltip },
+      });
+
+      expect(mockScrollIntoView).not.toHaveBeenCalled();
+      expect(showTooltip).not.toHaveBeenCalled();
+      expect(resetSeatJumpTo).not.toHaveBeenCalled();
```

Comment:
Should we assert also that the onClick is not called by default?

Reply:
I don't think that's needed here. I can't see any props that will trigger the `onClick` handler so if it does fire that'll be a bug in the implementation that should be very easy to spot.